pkg/api/controllers/profile.go

```diff
@@ -49,6 +48,40 @@ func (p *ProfilePortal) CreateProfile() {
 		return
 	}
 
+	//Validate StorageType is block or file
+	if profile.StorageType != "block" && profile.StorageType != "file" {
```

Comment:
should define constants for 'block' and 'file'

Reply:
if profile.StorageType != constants.Block && profile.StorageType != constants.File {
Done